### PR TITLE
DOCS-1987 Required NGINX versions

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/4.0/installation/nginx-plus.md [hunk 1]
```diff
@@ -19,15 +19,15 @@
 [install-postanalytics-instr]:      ../admin-en/installation-postanalytics-en.md
 [waf-installation-instr-latest]:     /installation/nginx-plus/
 [img-node-with-several-instances]:  ../images/user-guides/nodes/wallarm-node-with-two-instances.png
-[img-create-wallarm-node]:      ../images/user-guides/nodes/create-cloud-node.png
```
Comment (AnastasiaTWW) [inline docs/4.0/installation/nginx-plus.md:22]: is it deleted intentionally? is not it used?

File: include/waf/installation/nginx-distr-requirements-4.0.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+* Access to the account with the **Administrator** role in Wallarm Console for the [US Cloud](https://us1.my.wallarm.com/) or [EU Cloud](https://my.wallarm.com/)
```
Comment (AnastasiaTWW) [inline include/waf/installation/nginx-distr-requirements-4.0.md:1]: are you sure we should continue using includes for this content? is it still re-used in several places?

File: include/waf/installation/nginx-requirements-4.0.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 * Access to the account with the **Administrator** role in Wallarm Console for the [US Cloud](https://us1.my.wallarm.com/) or [EU Cloud](https://my.wallarm.com/)
 * SELinux disabled or configured upon the [instructions][configure-selinux-instr]
+* NGINX version 1.22.1.
```
Comment (AnastasiaTWW) [inline include/waf/installation/nginx-requirements-4.0.md:3]: extra dot in the end

File: include/waf/installation/nginx-requirements-4.0.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
...
+
+    !!! info "Custom NGINX versions"
+        If you have a different version, see [how to connect  the Wallarm module to custom build of NGINX][nginx-custom]
```
Comment (AnastasiaTWW) [inline include/waf/installation/nginx-requirements-4.0.md:6]: double space in the link text, please check

File: include/waf/installation/nginx-plus-requirements-4.0.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+* Access to the account with the **Administrator** role in Wallarm Console for the [US Cloud](https://us1.my.wallarm.com/) or [EU Cloud](https://my.wallarm.com/)
+* SELinux disabled or configured upon the [instructions][configure-selinux-instr]
+* NGINX Plus release 28 (R28).
```
Comment (AnastasiaTWW) [inline include/waf/installation/nginx-plus-requirements-4.0.md:3]: NGINX Plus VERSION...?

File: include/waf/installation/nginx-distr-requirements-4.0.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+* SELinux disabled or configured upon the [instructions][configure-selinux-instr]
+* NGINX version:
+
+    * 1.14.2 for Debian 10.x (buster)
+    * 1.18.0 for Debian 11.x (bullseye)
+    * 1.18.0 for CentOS 7.x
+    * 1.20.1 for AlmaLinux, Rocky Linux or Oracle Linux 8.x
```
Comment (AnastasiaTWW) [inline include/waf/installation/nginx-distr-requirements-4.0.md:8]: are these versions really different for OS?

File: include/waf/installation/nginx-requirements-4.0.md (hunk 3 above)
Comment (AnastasiaTWW) [inline include/waf/installation/nginx-requirements-4.0.md:3]: is the version the same for 4.4, 4.2, 4.0?

File: docs/4.0/installation/nginx-plus.md (hunk 1 above)
Comment (MaximBashurov) [reply docs/4.0/installation/nginx-plus.md:22]: Accidentally, restored.

File: include/waf/installation/nginx-distr-requirements-4.0.md (hunk 2 above)
Comment (MaximBashurov) [reply include/waf/installation/nginx-distr-requirements-4.0.md:1]: Yes, re-used in 3 files.

File: include/waf/installation/nginx-plus-requirements-4.0.md (hunk 5 above)
Comment (MaximBashurov) [reply include/waf/installation/nginx-plus-requirements-4.0.md:3]: NGINX Plus versions are officially called "releases": https://docs.nginx.com/nginx/releases/

File: include/waf/installation/nginx-requirements-4.0.md (hunk 3 above)
Comment (MaximBashurov) [reply include/waf/installation/nginx-requirements-4.0.md:3]: Yes.

File: include/waf/installation/nginx-distr-requirements-4.0.md (hunk 6 above)
Comment (MaximBashurov) [reply include/waf/installation/nginx-distr-requirements-4.0.md:8]: My understanding was that the essence of this task was to name versions explicitly, so how can we skip?

**Pull request merged**